test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -82,6 +84,42 @@ func AutomationConfigConfigMapExists(mdb *mdbv1.MongoDB) func(t *testing.T) {
 	}
 }
 
+// FeatureCompatibilityVersion verifies that the FeatureCompatibilityVersion is
+// set to `version`. The FCV parameter is not signaled as a non Running state, for
+// this reason, this function checks the value of the parameter many times, based
+// on the value of `tries`.
+func FeatureCompatibilityVersion(mdb *mdbv1.MongoDB, version string, tries int) func(t *testing.T) {
```

Comment:
maybe this could have a better name. `HasFeatureCompatibilityVersion` or `ReachesFeatureCompatibilityVersion`